config/rbac/assignmetadata_viewer_role.yaml

```diff
@@ -0,0 +1,20 @@
+# permissions for end users to view assignmetadata.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: assignmetadata-viewer-role
+rules:
+- apiGroups:
+  - mutations.gatekeeper.sh
+  resources:
+  - assignmetadata
```

Comment:
I'm curious why one resource is "assigns" (plural) and the other is "assignmetadata" (singular)

Also, I don't see the CRD manifests?

Reply:
assignmetadata changed to plural

Reply:
awesome, thanks!